cli_tools/osconfig_agent/main.go

```diff
@@ -42,22 +101,42 @@ func strIn(s string, ss []string) bool {
 	return false
 }
 
-func main() {
-	flag.Parse()
-	patchInit()
-
-	ctx := context.Background()
-
+func run(ctx context.Context) {
 	client, err := osconfig.NewClient(ctx, option.WithEndpoint(*endpoint), option.WithCredentialsFile(*oauth))
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalln("NewClient Error:", err)
 	}
 
-	res, err := lookupConfigs(ctx, client, *resource)
+	res, err := getResource(*resource)
```

Comment:
This is for lookup up the instance info right? If so, should it be called that instead?